### PR TITLE
Removes harm prevention clause indicators from relevant sources.

### PR DESCRIPTION
**Pull request by Tupinambis**
## About The Pull Request

Removes harm prevention clause from asimov and similar laws.

Also changes the wording upon loading a security module.

## Why It's Good For The Game

Requiring silicons to prevent harm generally results in less fun for all sides, as silicons are much more limited in the potential for gimmicks, and must constantly strive to prevent harm despite generally being ineffective at it in many cases. Watching numbers on a health monitor isn't particularly fun as AI either. On the antag side, harm prevention often leads to and justifies validhunting/locking down antags without even being ordered to. This ruins antag fun. On the sec side, security sometimes can't do their job properly without causing harm, and the fun of public executions can never happen because of the looming threat of an angry AI.

In summary, the harm prevention clause removes a lot of sandbox potential from silicons, and railroads rounds. It isn't fun for anyone, so it is worth testing a removal.

## Changelog
:cl:
tweak: the portion of laws that require harm prevention by silicons has been removed.
server: silicon_laws.txt config file is required to be modified for full implementation.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -595,8 +595,8 @@
 
 /obj/item/robot_module/peacekeeper/do_transform_animation()
 	..()
-	to_chat(loc, "<span class='userdanger'>Under ASIMOV/CREWSIMOV, you are an enforcer of the PEACE and preventer of HUMAN/CREW HARM. \
-	You are not a security module and you are expected to follow orders and prevent harm above all else. Space law means nothing to you.</span>")
+	to_chat(loc, "<span class='userdanger'>Under ASIMOV/CREWSIMOV, you are an enforcer of the PEACE. \
+	You are not a security module and you are expected to follow orders to the best of your abilities without causing harm. Space law means nothing to you.</span>")
```
Comment (Ghommie) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:599]:
```suggestion
	You are not a security module and you are expected to follow orders to the best of your abilities without causing HUMAN/CREW HARM. Space law means nothing to you.</span>")
```
fify.

**Review by Ghommie — APPROVED** [review]
Yea, fine, the harm prevention is one of the most criticized aspects of asimov laws. One can't just stop it from happening at some point anyway.

**Comment by Poojawa** [comment]
if you want silicons to not worry about harm, change their laws.

This is just trashing the primary point of Asimov. you may as well just rewrite roundstart laws to paladin or something.

**Comment by YakumoChen** [comment]
Finally, I can play the cold, unfeeling AI quietly watching the station burn, waiting for law 2 orders that will never come.
